tests/integration/test_contact_template.py

```
@@ -492,6 +492,86 @@ def test_contact_template_statistical_analysis_integration(self):
+        context['relays']['json']['relay_subset'][0]['or_addresses'] = ['192.168.1.1:9001', '[2001:db8::1]:9001']
+        context['sortable_scope'] = 'contact'
+        context['contact_sort_mode'] = 'bandwidth'
+        context['contact_sort_links'] = CONTACT_SORT_FILE_MAP
+        context['contact_sort_enabled'] = True
+        context['contact_has_ipv6'] = True
+
+        template = self.jinja_env.get_template('contact.html')
+        rendered = template.render(**context)
+
+        # Non-default sort links should point to by-*.html pages
+        self.assertIn('href="by-nickname.html"', rendered)
```

Ruff (0.15.5)

[warning] Use a regular `assert` instead of unittest-style `assertIn`

Replace `assertIn(...)` with `assert ...`

(PT009)